ast/builtins.go

```diff
@@ -800,6 +802,32 @@ var HTTPSend = &Builtin{
 	),
 }
 
+/**
+ * Set
+ */
+
+// Intersection returns the intersection of the given input sets
+var Intersection = &Builtin{
+	Name: "intersection",
+	Decl: types.NewFunction(
+		types.NewAny(
+			types.NewSet(types.NewSet(types.A)),
```

Comment:
The outer `types.NewAny` is redundant here, so let's remove it. Same with the union declaration below.

Reply:
Done